galaxy/tools/deps/resolvers/tool_shed_packages.py

```diff
@@ -24,9 +24,7 @@ def _find_dep_default( self, name, type='package', **kwds ):
             installed_tool_dependency = self._get_installed_dependency( name, type, version=None, **kwds )
             if installed_tool_dependency:
                 dependency = self._get_set_environment_installed_dependency_script_path( installed_tool_dependency, name )
-                is_galaxy_dep = isinstance(dependency, GalaxyPackageDependency)
-                has_script_dep = is_galaxy_dep and dependency.script and dependency.path
-                if has_script_dep:
+                if dependency.script and dependency.path:
```

Comment:
Probably this should be reverted and Pulsar version merged to Galaxy, this was last changed in commit e97a95a06d5cc0903125d9e46ab9bf33f7759091 .